runner/utils/driver/istio/istio.go

```diff
@@ -66,7 +76,41 @@ func NewIstioDriver(auth *auth.Module, c *Config) (*Istio, error) {
 	// Start the keda external scaler
 	go kedaScaler.Start()
 
-	return &Istio{auth: auth, config: c, kube: kube, istio: istio, keda: kedaClient, kedaScaler: kedaScaler}, nil
+	waitservice := make(map[string]deployments)
```

Comment:
Remove this variable, create the istio struct here & assign the struct to a variable called i

Use the the i (i.servicestatus) variable in the watchDeployments function